### PR TITLE
Changing the config key as suggested by pamil #190

### PR DESCRIPTION
**Pull request by mamazu**
THIS IS A BC BREAK and needs to be documented. (How?)

### CODE REVIEW

**Comment by lchrusciel** [comment]
Fixing #263, probably we should document some UPGRADE.md file

**Comment by lchrusciel** [comment]
Also, it should be adjusted at least [here](https://github.com/Sylius/ShopApiPlugin#attributes)

**Comment by mamazu** [comment]
> Fixing #263, probably we should document some UPGRADE.md file

But for which version?

**Comment by lchrusciel** [comment]
We need to create a new record in https://github.com/Sylius/ShopApiPlugin/blob/master/UPGRADE.md, and notice all the changes, that are BC-Breaks. Problem is that we are over 200 commits from the last release. Nevertheless, some changes that are pretty straightforward to describe. 

(just to keep a track #340)

**Comment by mamazu** [comment]
Can you generate a list of all the commit messages and I will try to craft a `UPGRADE.md` that reflects those changes.

File: UPGRADE.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# UPGRADE FROM 1.0.0-beta.18 to 1.0.0-beta.20
```
Comment (lchrusciel) [inline UPGRADE.md]:
```suggestion
# UPGRADE FROM 1.0.0-beta.21 to 1.0.0-beta.22
```

**Comment by lchrusciel** [comment]
Thank you, @mamazu! :tada:

**Pull request merged**